js/.bowerrc

```diff
@@ -1,4 +1,5 @@
 {
+  "registry": "https://registry.bower.io",
```

Comment:
Can you please help review https://github.com/nextcloud/calendar/pull/859 instead? :) 
THX!